MessagePrompt.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+
+namespace CMP1124M_Assignment
+{
+    public class MessagePrompt
+    {
+        public string Message { get; private set; }
+        public MessagePrompt(string message)
+        {
+            Message = message;
+        }
+
+        public void ShowMessage()
+        {
+            Console.WriteLine(Message);
+        }
+
```

Comment:
Should add feature to be able to select option(s) rather than simply specifying a number